src/converse-chatview.js

```diff
@@ -67,6 +67,7 @@ converse.plugins.add('converse-chatview', {
                 'emoji': true,
                 'spoiler': true
             },
+            'image_urls_regex': null
```

Comment:
When adding a new configuration option, please also document it here:
https://github.com/conversejs/converse.js/blob/master/docs/source/configuration.rst

Options are in alphabetical order.

Also, please provide a test for this. In the test you can pass in a regex for `image_urls_regex` and test that images render.

Your test will be something similar to this:
https://github.com/conversejs/converse.js/blob/823134d7629bad2b82875964169340609c0ce2c9/spec/messages.js#L937

Reply:
Done.